package.json

```diff
@@ -25,6 +26,7 @@
     "netlify-cms-app": "^2.15.72",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-helmet": "^6.1.0",
```

Comment:
With the new Gatsby Head API I don't think we'll need react-helmet for this project.

Reply:
Ok, I'll take that out later today. It's only there cause the docs said to install it. So long as it works without it, it should be fine.